hardware/RF-Test-Guide.md

```diff
@@ -0,0 +1,283 @@
+In order to get an optimized radiation performance, the antenna might require tuning that can be realized in two ways:
+- Dimension changes in the antenna layout structure;
+- Applying external tuning components.
+
+Typically it prefer to the solution that the layout modification is not required in a custom design. To accomplish this, Silicon Labs generally recommends to use SMD placeholders as external antenna tuning components, which the suggested external antenna matching structure is a 3-element PI network as following picture shows. You can achieve a good match using as a maximum of two elements (with one series and one shunt component) of the PI network. Any unknown passive impedance can be matched to 50 ohms by using this PI network, since all L, C, L-C, C-L combinations can be realized on it, and therefore any de-tuning effect can be compensated out.
+
+Note that every implementation of an antenna design might require different combinations of inductors and capacitors.
+
+The recommended 3-element PI network for external antenna matching purposes is here:
+
+<div align="center">
+  <img src="files\HW-RF-Test-Guide\matching.png">
+</div>
+
+<div align="center">
+
+<b>Figure 1.  3-element PI type matching network</b> 
+
+</div>
+
+
+## 2.2. Tune and measure
+
+It is recommended to add a pi network for antenna matching as shown in Figure 1. Finding appropriate values for the antenna matching components should be considered an iterative task. Usually it use a Vector Network Analyser(VNA) to tune and measure the antenna matching network as following figure shows. 
+
+<div align="center">
+ <img src="files\HW-RF-Test-Guide\VNA-Test.png">  
+</div>
+
+The following matching steps and tricks are proposed:
+
+- Calibrate your VNA for a frequency range larger than the intended bandwidth of the antenna.
+- Connect an RF coaxial cable to the RF line (for instance by soldering a pigtail to the line). Connect the RF coaxial cable to a VNA to measure the reflection coefficient, S11, looking into the antenna through the matching network.
+
+  a. Be sure to have a good connection to the ground plane to get the best electrical performance and the highest mechanical robustness during the measurement.
+
+  b. Make sure to route the pigtail towards the center of the PCB and then perpendicularly away from the PCB at the center point. This will limit the effect of the cable on the measured data as much as possible.
+
+- Start with no components on the antenna network shown in Figure 1.
+
+  a. The shunt components are not mounted.
+
+  b. The series component is not mounted.
+
+- Using port extension function in the VNA to move the reference point to the footprint of R1 and R2. This is achieved when the locus of the S-parameters in the Smith chart on the VNA have assembled in a point at the right edge of the Smith chart.
```

Comment:
Would a smith chart image be helpful to include here?

Reply:
Added the illustrate picture.